nextcloudappstore/core/forms.py

```diff
@@ -50,27 +51,40 @@ class AppRegisterForm(Form):
                               'openssl base64</b>'))
     safe_help_fields = ['certificate', 'signature']
 
+def get_languages_local(language=None):
+    if language:
+        languages = list(language)
+    else:
+        languages = [ l[0] for l in settings.LANGUAGES]
```

Comment:
Yes, see https://github.com/nextcloud/appstore/pull/414#discussion_r94726283 where I'm getting the local name to display in the choice field.